### PR TITLE
fix(svg): generate unique IDs for gradients based on their properties

### PR DESCRIPTION
**Pull request by j3j5**
As discussed on #186

### CODE REVIEW

**Comment by j3j5** [comment]
I'v added 2 simple tests for the SVG renderer.

**Comment by DASPRiD** [comment]
LGTM!

File: test/Integration/SVGRenderingTest.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        $writer = new Writer($renderer);
+
+        $svgCode = $writer->writeString('Hello World!');
+        $expected = file_get_contents(__DIR__.'/__snapshots__/files/SVGRenderingTest__testGenericQrCode__1.svg');
+
+        $this->assertEquals($expected, $svgCode);
+    }
+
+    //    SVGRenderingTest__testQrWithGradientGeneratesDifferentIdsForDifferentGradients_horizontal
+    //SVGRenderingTest__testQrWithGradientGeneratesDifferentIdsForDifferentGradients_vertical
```
Comment (j3j5) [inline test/Integration/SVGRenderingTest.php:37]: Forgot these two comments here, feel free to remove them @DASPRiD

**Comment by vlakoff** [comment]
Nit: the newer `xxh3` would be a better choice than `xxh64` — it's even faster while keeping the same 64‑bit width.

**Pull request merged**